examples/cargo_without_cmake/Cargo.toml

```diff
@@ -20,8 +20,11 @@ license = "MIT OR Apache-2.0"
 [dependencies]
 cxx = "1.0.75"
 cxx-qt = { path = "../../crates/cxx-qt" }
+# Use `cxx-qt = "0.4"` here instead!
 cxx-qt-lib = { path = "../../crates/cxx-qt-lib" }
+# Use `cxx-qt-lib = "0.4"` here instead!
```

Comment:
should these be above or below the line they are replacing? :thinking:

Reply:
Hm, it kind off doesn't matter, but above does read somewhat nicer.
Changed.